Editor/SimpleGraphQL/GraphQLImporterV1.cs

```diff
@@ -16,30 +17,37 @@ public override void OnImportAsset(AssetImportContext ctx)
         {
             var lexer = new Lexer();
             var parser = new Parser(lexer);
+            var regex = new Regex("#import \\\"(?<path>[\\w\\/.]+)\\\"");
             string contents = File.ReadAllText(ctx.assetPath);
+            foreach(Match match in regex.Matches(contents))
+            {
+                var replacementContent = File.ReadAllText(Path.GetFullPath(Path.Combine(Path.GetDirectoryName(ctx.assetPath), match.Groups["path"].Value)));
```

Comment:
This is a little difficult to read, not sure if it's worth splitting some of it out into variables so the intention is clearer.